src/schema/v2/conversation/deleteConversationMutation.ts

```diff
@@ -0,0 +1,42 @@
+import { GraphQLString, GraphQLNonNull } from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import Conversation from "schema/v2/conversation"
+import { ResolverContext } from "types/graphql"
+
+interface DeleteConversationMutationInputProps {
+  id: string
+  spam: boolean
+}
+
+export default mutationWithClientMutationId<
+  DeleteConversationMutationInputProps,
+  any,
+  ResolverContext
+>({
+  name: "DeleteConversationMutation",
+  description: "Delete a conversation.",
+  inputFields: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The id of the conversation to be deleted.",
+    },
+  },
+  outputFields: {
+    conversation: {
+      type: Conversation.type,
+      resolve: (conversation) => conversation,
+    },
+  },
+  mutateAndGetPayload: async (args, { conversationDeleteLoader }) => {
+    if (!conversationDeleteLoader) {
+      return null
+    }
+
+    try {
+      const response = await conversationDeleteLoader(args.id)
+      return response
+    } catch (error) {
+      throw new Error(JSON.parse(error.body).error)
```

Comment:
For some reason this endpoint returns stringified JSON as an error that needs to be parsed. Will add a comment here / investigat

Reply:
This should be using the `orError` pattern?